packages/coding-standard/src/Rules/AbstractPrefferedCallOverFuncRule.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Symplify\CodingStandard\Rules;
+
+use PhpParser\Node;
+use PhpParser\Node\Expr\FuncCall;
+use PHPStan\Analyser\Scope;
+use PHPStan\Reflection\MethodReflection;
+use ReflectionClassConstant;
+use ReflectionException;
+use Symplify\CodingStandard\PhpParser\NodeNameResolver;
+use Symplify\SymplifyKernel\Exception\ShouldNotHappenException;
+
+abstract class AbstractPrefferedCallOverFuncRule extends AbstractSymplifyRule
+{
+    /**
+     * @var array<string, string[]>
+     */
+    private $funcCallToPrefferedStaticCalls = [];
+
+    /**
+     * @var NodeNameResolver
+     */
+    private $nodeNameResolver;
+
+    /**
+     * @param array<string, string[]> $funcCallToPrefferedStaticCalls
+     */
+    public function __construct(NodeNameResolver $nodeNameResolver, array $funcCallToPrefferedStaticCalls = [])
```

Comment:
The `$funcCallToPrefferedStaticCalls` name is incorrect for method call rule

Reply:
updated argument name to `$funcCallToPrefferedCalls`

Reply:
As the rules have different names, the parameters should too, to prevent miss-placing one with another.

- `$funcCallToPrefferedStaticCalls`
- `$funcCallToPrefferedMethodCalls`

Reply:
implemented.